apps/namehashlabs.org/components/2 - molecules/mobile-menu.tsx

```diff
@@ -50,13 +56,14 @@ const MobileMenu = () => {
             >
               <NameHashLabsLogo className="text-white" />
             </Link>
-            <button
-              className="p-3 hover:bg-white hover:bg-opacity-10 rounded-[6px] transition-colors duration-200"
+            <IconButton
```

Comment:
@eduramme Hey I was testing this but for some reason it seems the hover state isn't working for me here?

![CleanShot 2024-09-03 at 18 29 20](https://github.com/user-attachments/assets/f6ed9a47-bd58-4f91-8217-08ce80f9e250)

The other buttons in this mobile menu had their hover states working well.